code-challenges.test.js

```diff
@@ -24,13 +24,30 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
+//use .length
 
 
 
 // b) Create the function that makes the test pass.
-
-
+describe("when a number greater than 2 is called", () => {
+  it("returns an array of the length containing the Fibonacci sequence", () => {
+    //arrange
+    const myArray = function (a)
```

Comment:
to allow this function to be reusable, you would need to declare it outside of the describe-block